### DISCUSSION TITLE
Quick install problem

### DISCUSSION THREAD

**Discussion by Ikkow** [inventree/InvenTree#6519] [Q&A] [answered]
Hello, I need your help when I launch the InvenTree quick install I get this :
![image](https://github.com/inventree/InvenTree/assets/153614914/a48700a4-2307-4b97-a3f6-f6ee0b2cb25f)
Any ideas ?

**Comment by matmair** [comment, marked as answer]
It is right in the error messages: your system is missing the curl command which leads to a missing gpg key.

Please open an installation bug report the next time, I can only guess with the provided info and screenshots are not accessible without extra tools.

**Reply by Ikkow** [reply]
Okay thanks and sorry I'll do better next time

**Reply by matmair** [reply]
@Ikkow did you get it working?

**Comment by Mayurveer05** [comment]
> Hello, I need your help when I launch the InvenTree quick install I get this :
> ![image](https://github.com/inventree/InvenTree/assets/153614914/a48700a4-2307-4b97-a3f6-f6ee0b2cb25f)
> Any ideas ? 

Manually add Keys 
Then sudo apt update &&upgrade 
And then try again
